src/riak_kv_put_fsm.erl

```diff
@@ -46,6 +46,7 @@
          waiting_local_vnode/2,
          waiting_remote_vnode/2,
          postcommit/2, finish/2]).
+-export([get_hooks/2, get_hooks/3]).
```

Comment:
Dont think these functions need to be exported. I like the changes to seperate the Bucket out of the Bkey for the logic to select the hook but get_hook is only called within this module. Any reason i'm missing to make it public?